config/release-package-matrix.d.ts

```diff
@@ -8,6 +8,7 @@ export interface PackageInformation {
   name: string;
   description: string;
   packageOn: string;
+  packageType: 'dmg' | 'tar.gz' | 'deb' | 'rpm' | 'zip' | 'msi';
```

Comment:
`packageType` is already a function of `getDistro(name)`, I don’t think we need to separately store this here? For example, we distinguish between `win32` and `win32msi` already. (See also https://github.com/mongodb-js/mongosh/blob/main/packages/build/src/packaging/package/get-package-file.ts where we compute the actual filename based on the “getDistro()” value)

Reply:
I've seen the function, and the first implementation used it, but then I realised that I would need to get the extension from the filename, which is additional logic, and we already use getDistro like for a lot of different things. While I would agree that reusing code would be good practice, I preferred to be more explicit.

Do you still think that it would be better to use getDistro? I don't mind going back to the old implementation, and it is already under test, so it should be straightforward to change in case you strongly prefer to use getDistro.

Reply:
I've seen the function, and the first implementation used it, but then I realised that I would need to get the extension from the filename, which is additional logic, and we already use getDistro like for a lot of different things. While I would agree that reusing code would be good practice, I preferred to be more explicit.

Do you still think that it would be better to use getDistro? I don't mind going back to the old implementation, and it is already under test, so it should be straightforward to change in case you strongly prefer to use getDistro.

Reply:
Convinced, will do the change 👍